pennylane/_queuing.py

```diff
@@ -118,8 +118,12 @@ def remove(cls, obj):
 
     @classmethod
     def update_info(cls, obj, **kwargs):
-        """Updates information of an object in the queue."""
-        cls.active_context().update_info(obj, **kwargs)
+        """Updates information of an object in the active queue."""
+        cls.active_context()._update_info(obj, **kwargs)  # pylint: disable=protected-access
```

Comment:
Still not 100% sure we need this method in the parent class.
@josh146 do you remember why this was added to the `QueuingContext` instead of being left as a method of the inheriting `AnnotatedQueue`?